packages/ast/src/clients/helper-funcs/msg-funcs.ts

```diff
@@ -13,16 +22,61 @@ import { GenericParseContext } from "../../encoding";
 export function createMsgHelperCreator(
     context: GenericParseContext,
     service: ProtoService,
-    methodKey: string,
-    helperCreatorName: string
+    methodKey?: string,
+    helperCreatorName?: string
 ) {
-    // get related encoders.
-    // e.g.: encoders: toEncoders(MsgMultiSend),
+    console.log(`🍀 \n | 🍄 file: msg-funcs.ts:19 \n | 🍄 service:`, service);
+    const createrFunctions = Object.keys(service.methods).map((method) => {
+        const callExpression = ast.callExpression(ast.identifier("buildTx"), [
+            ast.objectExpression([
+                ast.objectProperty(
+                    ast.identifier("getSigningClient"),
+                    ast.identifier("getSigningClient")
+                ),
+                ast.objectProperty(
+                    ast.identifier("typeUrl: "),
+                    ast.memberExpression(
+                        ast.identifier(`Msg${method}`),
+                        ast.identifier("typeUrl")
+                    )
+                ),
+                //**TODO: Improvements, where can i grab toEncoders and toConverters instead of hard coding the string?
+                //!!FIX: the toEncoders and toConverters are functions calls, not strings
+                ast.objectProperty(
+                    ast.identifier("encoders"),
+                    ast.stringLiteral(`toEncoders(Msg${method})`)
+                ),
+                ast.objectProperty(
+                    ast.identifier("converters"),
+                    ast.stringLiteral(`toConverters(Msg${method})`)
+                ),
+            ]),
+        ]);
```

Comment:
need some guidance on how to write this better in AST

Reply:
This is tied to how to get the imports from UTILS in telescope package correctly.

Reply:
@Zetazzz can you share some insights on this? 

From CCA, I need to add this import { toConverters, toEncoders } from "@interchainjs/cosmos/utils";

But not sure how to add this to UTILS in telescope.

Reply:
after considering on this, we're going to import interchainjs anyway.

so please add @interchainjs/cosmos to dev depedencies and gen the import using the way of addUtil